src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -0,0 +1,74 @@
+import { getCLS, getFID, getLCP, getFCP, getTTFB } from 'web-vitals';
+
+// Stops ts assuming json values are null
+const temp: any = null;
+
+const jsonData = {
+	page_view_id: '',
+	received_timestamp: `${new Date().toISOString()}`,
+	id: 'prod',
+	received_date: new Date().toISOString().slice(0, 10),
+	fid: temp,
+	cls: temp,
+	lcp: temp,
+	fcp: temp,
+	ttfb: temp,
+};
+
+export const coreVitals = (): void => {
+	type CoreVitalsArgs = {
+		name: string;
+		value: number;
+	};
+
+	const getValue = ({ name, value }: CoreVitalsArgs): void => {
```

Comment:
Is `getValue` the right name here? From reading this function it's not returning anything and seems to instead be sending something?